examples/pbsam-barn_bars/barn_bars_electro.in

```diff
@@ -1,18 +1,18 @@
-g20read
+read
     mol pqr barstar.pqr
     mol pqr barnase.pqr
-
 end
+
 elec name comp_solv        # Gly
-    pbsam
+    pbsam-auto
 
     runtype electrostatics  # Can be energyforce, electrostatics etc
     runname elec_barn_bars  # Output name for files created by program
 
     units kT
     ## info for molecule
     tolsp 4.5
-    mesh skin  #comment if the system has been coarse-grained
+    # mesh skin  #comment if the system has been coarse-grained
```

Comment:
@sobolevnrm should `mesh` be part of the `pbsam-auto` specification? It is also not documented.

Reply:
I honestly don't know...

Reply:
I will create an issue for this but who should we ask to find out? Or do I just need to dig through the code and try to figure out what is used and not used?